README.md

```diff
@@ -1 +1,214 @@
-# Catalog-of-my-things
\ No newline at end of file
+<a name="readme-top" align="center"></a>
+
+<div align="center">
+  <h1>Catalog Of My Things</h1>
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) --> 
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog Of My Things <a name="about-project"></a>
+In this project, my partners [Elarh](https://github.com/elarhadu),[Yuvenal](https://github.com/yuvenalmash) and I created a console app that helps users to keep a record of different types of things they own: books, music albums, movies, and games. Everything is based on the UML class diagram presented below. The data will be stored in JSON files.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Add a Book
+- Add a Game
+- Create and manage data about my things
+- Add a Music Album
+- List all added items
+- Keep track of books, music albums, movies, and games and other information
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+LIVE DEMO 
+
+## 🚀 Demo Video <a name="live-demo"></a>
+Please visit our zoom video presentation. Click the [Link](https://youtu.be/sAieEwEXhl4) button.
```

Comment:
- [OPTIONAL] Kindly note that presentation is a core part of soft skills, I love that everyone was audible  in the presentation 👏  but moving forward @elarhadu please let the light/light soucre be in front of you in calls or presentations so that your face is visible to the viewers because you weren't visible on the presentation.